blog/2025-2-15-pixi-v8-launches.md

```diff
@@ -0,0 +1,228 @@
+
+It's hard to believe that PixiJS has been part of the open-source community for a whopping ten years. In that time, the digital landscape has evolved tremendously, and so has PixiJS. We've seen significant updates, like the transition to TypeScript, and we've overhauled major parts of the engine, such as asset loading and WebGL integration.
+
+Now, we're thrilled to unveil PixiJS v8, arguably our most substantial update ever. This release is not just a reflection on the shortcomings of v7, which has served us well, but an acknowledgment that there's always room for improvement. Over time, we've all encountered aspects of our code we wished we could refine. Often, the best solutions and insights emerge only after we've stepped back from the problem, allowing us to see the bigger picture.
+
+With PixiJS v8, our aim was to revisit and enhance the foundation of PixiJS, streamlining its core rather than just adding layers of code.
+
+Our vision for v8 was clear:
+- **Longevity:** We designed v8 to stand the test of time, anticipating it will remain relevant and robust for another decade.
+- **Innovation with WebGPU:** Embracing the latest in rendering technology, we've seamlessly integrated WebGPU, not as an add-on to our existing WebGL renderer but as a core paradigm, ensuring PixiJS remains at the cutting edge as WebGL phases out.
+- **Leveraging Modern JavaScript:** The advancements in JavaScript have significantly simplified development. We've utilized features like object destructuring and options to make v8 cleaner and more powerful.
+- **Correcting Past Oversights:** Every project has its lessons. With v8, we've addressed and rearchitected certain aspects of PixiJS, reducing complexity and enhancing functionality, particularly in areas we felt were overengineered in the past (looking at you, textures!).
+- **Boosting Performance:** PixiJS is already renowned for its speed. With v8, we've unlocked even greater performance, making it faster across the board compared to v7.
+
+We're incredibly proud of PixiJS v8 and eager to share the improvements and new features with you. While there are some breaking API changes, we've provided a migration guide and ensured compatibility with v7 wherever possible. Get ready to experience the next level of 2D rendering with PixiJS v8!
+
+---
+
+## 🔗 Quick links
+- The new Docs for v8 can be found here [LINK TO DOCS]
+- [Migration](https://pixijs.com/guides/migrations/v8)
+- [Examples](https://pixijs.com/examples)
+- [Open Games](https://github.com/pixijs/open-games)
+
+---
+
+## 🎁 Whats New?
+
+So many new things - more than we can easily cover in one post! Here's some head lines, but check out the links above for a more in depth details on the changes!
+#### 📦 New Package Structure
+
+No more "lerna." PixiJS is now just one package with one import root: `import {stuff} from ‘pixi.js’`. This change means we now have much better tree shaking during app compilation, reducing bundle size if not imported.
```

Comment:
We kinda abandoned Lerna awhile ago. I think the thing to celebrate here is the complexity of moving away from publishing multiple packages and all the npm headaches, linking locally, and ecosystem issues that caused.
```suggestion
No more multiple packages (e.g., `@pixi/stuff`). PixiJS is now just one package with one import root: `import {stuff} from ‘pixi.js’`. This change means we now have much better tree shaking during app compilation, reducing bundle size if not imported.
```

Reply:
Also, we do have submodule exports, so you you can do `pixi.js/unsafe-eval` for example for optional things. Not sure if the submodules are worth calling out?